apps/src/blockly/constants.js

```diff
@@ -70,13 +70,20 @@ export function stringIsXml(str) {
   }
 }
 
+export const BLOCK_TYPES = {
```

Comment:
@molly-moen Thoughts on adding a constant we can use to import these strings and what it should be called? `BLOCK_TYPES` seems possibly too generic because, for example, Music Lab has specific block types in its own subfolder (see https://github.com/code-dot-org/code-dot-org/blob/344e2576ff3252a9883689fbbe0b27daee116e62/apps/src/music/blockly/blockTypes.ts), but I couldn't come up with a better name.

Reply:
`BLOCK_TYPES` makes sense to me. I think it's ok for music lab to have a duplicate since it's in a clear `music` folder.